src/quote/flashmint/hyeth/component-quotes/index.ts

```diff
@@ -37,6 +38,13 @@ export class ComponentQuotesProvider {
     )
   }
 
+  isMorpho(token: string) {
+    return isAddressEqual(
+      token as Address,
+      '0x78Fc2c2eD1A4cDb5402365934aE5648aDAd094d0'
```

Comment:
one thought, not for this PR: we should export a constant for our tokens in tokenlists, so we can reference them directly without needing find logic...e.g. this could become: `INDEX_TOKENS.HYETH.address`